fcp/src/containment/structures/enums/data.py

```
@@ -0,0 +1,16 @@
+from enum import Enum, EnumMeta
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)